auth_otp/models.py

```
@@ -1,3 +1,438 @@
+        blank=True,
+        verbose_name='دلیل'
+    )
+    
+    expires_at = models.DateTimeField(
+        verbose_name='زمان انقضا'
+    )
+    
+    class Meta:
+        verbose_name = 'توکن مسدود'
+        verbose_name_plural = 'توکن‌های مسدود'
+        ordering = ['-blacklisted_at']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)